### PR TITLE
SDKS-2948 crash fix and improvement

### PR DESCRIPTION
**Pull request by jeyanthanperiyasamy**
# JIRA Ticket

Please link jira ticket here

# Description

Briefly describe the change and any information that would help speedup the review and testing.

### CODE REVIEW

File: forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/BiometricBindingHandler.kt [hunk 1]
```diff
@@ -11,10 +11,8 @@ import androidx.biometric.BiometricPrompt.AuthenticationCallback
 import androidx.biometric.BiometricPrompt.CryptoObject
```
Comment (rodrigoareis) [inline forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/BiometricBindingHandler.kt:11]: Update copyright

File: forgerock-core/src/main/java/org/forgerock/android/auth/biometric/BiometricAuth.kt [hunk 2]
```diff
@@ -70,7 +70,7 @@ class BiometricAuth @JvmOverloads constructor(
      */
     private val description: String? = null,
 
-) {
+    ) {
```
Comment (rodrigoareis) [inline forgerock-core/src/main/java/org/forgerock/android/auth/biometric/BiometricAuth.kt:73]: Update copyright

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
Changes looks good. Just need to update the copyright

**Comment by jeyanthanperiyasamy** [comment]
Addresses the comments

**Review by spetrov — APPROVED** [review]
Looks good to me!

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Pull request merged**